airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -95,6 +95,9 @@ def get_connection_form_widgets() -> dict[str, Any]:
             ),
             "extra__kubernetes__disable_verify_ssl": BooleanField(lazy_gettext('Disable SSL')),
             "extra__kubernetes__disable_tcp_keepalive": BooleanField(lazy_gettext('Disable TCP keepalive')),
+            "extra__kubernetes__sidecar_container_image": StringField(
```

Comment:
I  would like to suggest to make the name more explicit, like `extra__kubernetes__xcom_sidecar_container_image`, since it's possible to add other "sidecar" container(s) to the Pods.

Does this make sense to you?

Reply:
Done